### PR TITLE
Implement `kube_namespace:` prefix in `ac_include` and `ac_exclude`

### PR DESCRIPTION
**Pull request by L3n41c**
### What does this PR do?

Implement `kube_namespace:` prefix in `ac_include` and `ac_exclude`

### Motivation

`ac_include` and `ac_exclude` can be used to collect logs and metrics based on container name and image name.
However, some customers want to be able to discard a whole namespace.

### CODE REVIEW

**Review by clamoriniere — DISMISSED** [review]

File: releasenotes/notes/namespace_include_exclude-52ef2afcf7b7bdb2.yaml
Comment (olivielpeau) [inline releasenotes/notes/namespace_include_exclude-52ef2afcf7b7bdb2.yaml:11]: typo in option names + case + some language change :) :

```suggestion
    The ``ac_include`` and ``ac_exclude`` auto-discovery parameters now support the ``kube_namespace:`` prefix to collect or discard logs and metrics for whole namespaces in addition to the ``name:`` and ``image:`` prefixes to filter on container name and image name.
```

File: pkg/collector/python/containers.go
Comment (olivielpeau) [inline pkg/collector/python/containers.go:29]: Although this function is quite simple, I think it's worth adding a test for it now (in particular a non-regression test to make sure it will always handle a `nil` namespace gracefully)

(note: adding tests for these functions is a bit weird, it requires one level of indirection, see kubeutil_test.go for example)

**Review by olivielpeau — DISMISSED** [review]
I reviewed the python bindings part in particular. LGTM once my comments are addressed (no need for further approvals from me)

Made a couple of inline nits. Also, can you update this docstring? https://github.com/DataDog/datadog-agent/blob/1b51b001aa2008d11a63c30b95c3148317d2c553/rtloader/common/builtins/containers.c#L17

**Review by clamoriniere — DISMISSED** [review]

File: pkg/collector/python/containers.go
Comment (L3n41c) [reply pkg/collector/python/containers.go:29]: Test added.
Thanks for your review !

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**
